init_autoloader.php

```diff
@@ -24,3 +24,7 @@
 if (!class_exists('Zend\Loader\AutoloaderFactory')) {
     throw new RuntimeException('Unable to load dependencies. Run `php composer.phar install`.');
 }
+
+//register doctrine extensions
+$classLoader = new \Doctrine\Common\ClassLoader('Numeric', __DIR__.'/extensions/doctrine');
+$classLoader->register();
```

Comment:
Ik weet niet precies hoe we dit het beste zouden kunnen doen binnen de module. Zend framework heeft wel een mooie autoloader ingebouwd per module, maar die gaat hiervoor niet werken.